src/huggingface_hub/README.md

```diff
@@ -113,9 +119,19 @@ huggingface-cli whoami
 huggingface-cli repo create
 ```
 
-We also have an API to query models and datasets by specific tags (e.g. if you
-want to list models compatible to your library). Look at `list_models()`,
-`model_info()`, `list_datasets()`, and `dataset_info()`. 
+With the `HfApi` class there are methods to query models, datasets, and metrics by specific tags (e.g. if you want to list models compatible with your library):
+- **Models**:
+  - `list_models()`
+  - `model_info()`
+- **Datasets**:
+  - `list_datasets()`
+  - `dataset_info()`
+- **Metrics**:
+  - `list_metrics()`
+  - `metric_info()`
```

Comment:
I'd not mention metrics

There is the same API for Spaces though, i'd rather implement/document that one